portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,48 +14,37 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.sps.data.CommentsManager;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
+import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> comments = new ArrayList<>();
   private static final String COMMENT_INPUT = "comment-input";
-
+  private static final Gson GSON = new Gson();
```

Comment:
GSON variable should be lowercase, as it is not a constant, and remove the final even though it is possible to do that

Reply:
okay!